app/views/layouts/application.html.erb

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
 <html>
```

Comment:
- [ ] It would be better if we maintain a professional repo by adding the title, description, how to run locally, etc